bin/crew

```diff
@@ -1798,6 +1806,31 @@ def search_command(args)
   end
 end
 
+def sysinfo_command(args)
+  # newer version of Chrome OS exports info to env by default
+  if File.exist?('/etc/lsb-release')
+    lsb_release = File.read('/etc/lsb-release').scan(/^(.+?)=(.+)$/).to_h
+  else
+    # newer version of Chrome OS exports info to env by default
+    lsb_release = ENV
+  end
+
+  puts <<~MD
+    - Architecture: `#{ARCH_ACTUAL}` (`#{ARCH}`)
+    - Kernel version: `#{`uname -r`.chomp}`
```

Comment:
Shouldn't need to nest the backticks.  This should work too?  `\`uname -r\`.chomp`

Reply:
The outermost backticks are for markdown styling (`test`)